ultrasonic/src/main/kotlin/org/moire/ultrasonic/playback/CachedDataSource.kt

```diff
@@ -27,7 +27,7 @@ import timber.log.Timber
 @androidx.annotation.OptIn(androidx.media3.common.util.UnstableApi::class)
 class CachedDataSource(
     private var upstreamDataSource: DataSource
-) : BaseDataSource(false) {
+) : BaseDataSource(true) {
```

Comment:
We should check what this does in the Media3 code, but there is a possibility for this to go to the network so I think it is better to signal that